sdk/src/main/java/com/mapbox/maps/renderer/RendererError.kt

```diff
@@ -0,0 +1,49 @@
+package com.mapbox.maps.renderer
+
+import com.mapbox.maps.MapView
+import javax.microedition.khronos.egl.EGL10
+
+/**
+ * Typed enum wrapping up all the EGL error codes represented as ints.
+ * We do have SDK specific error types: [NO_VALID_EGL_CONFIG_FOUND] and [OUT_OF_MEMORY].
+ */
+class RendererError(val eglErrorCode: Int) {
+  /**
+   * Indicates whether some other object is "equal to" this one.
+   */
+  override fun equals(other: Any?): Boolean = other is RendererError &&
```

Comment:
will be nice to add [EqualsHashCodeTest](https://github.com/mapbox/mapbox-maps-android/blob/main/extension-style/src/test/java/com/mapbox/maps/extension/style/EqualsHashCodeTest.kt)

Reply:
thanks for reminding, will add